### PR TITLE
Yospace 3.6.0

### PR DESCRIPTION
**Pull request by saravanans-github**
upgraded 
1. Yospace to `v3.6.0`
 1.1 updated code references
2. BitmovinPlayer to `v8.114.0`

HLS - PASSED Yospace validation on Mac/Chrome
Yospace-Validation-Passed

### CODE REVIEW

**Comment by saravanans-github** [comment]
@dweinber kindly review PR for yospace upgrade to v3.6.0

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "lint": "eslint --quiet src/",
     "format": "prettier --write src/",
     "clean": "rm -R ./dist",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "3.6.0": "rm -rf node_modules; npm i -S @yospace/admanagement-sdk@3.6.0; npm i -S bitmovin-player@8.114.0; npm install"
```
Comment (dweinber) [inline package.json:19]: This doesn't look like it should be committed. I assume just an artifact from local testing?

**Review by dweinber — CHANGES_REQUESTED** [review]
Looks good, just one small remark regarding the package.json, rest looks good 👍 Thanks for the work!

File: package.json (hunk 1 above)
Comment (saravanans-github) [reply package.json]: ah indeed. sorry removed.

**Review by dweinber — APPROVED** [review]

**Pull request merged**